packages/whenable/src/track-turns.js

```diff
@@ -0,0 +1,113 @@
+/* global globalThis */
```

Comment:
Likewise, I diffed and found this is identical to the one in @endo/eventual-send. Could you just import from that one rather than duplicating?

Reply:
Can't.  `packageJson.exports` forbids.

Reply:
Understood. To be addressed later, probably after migrating whenable (vow) to endo. Resolving.